File: 09/VB/09CustomControls/Client/UserCode/PasswordBoxControl901.vb

Namespace LightSwitchApplication

    Public Class PasswordBoxControl901

        Private Sub PasswordBoxControl901_InitializeDataWorkspace(ByVal saveChangesTo As Global.System.Collections.Generic.List(Of Global.Microsoft.LightSwitch.IDataService))
            ' Write your code here.
            Me.UserProperty = New User()
        End Sub

        Private Sub PasswordBoxControl901_Saved()
            ' Write your code here.
            Me.Close(False)
            Application.Current.ShowDefaultScreen(Me.UserProperty)
        End Sub

        Private Sub PasswordBoxControl901_Activated()
            ' Write your code here.
            Dim password As IContentItemProxy = Me.FindControl("Password")
            password.SetBinding(System.Windows.Controls.PasswordBox.PasswordProperty, "Value",
                Windows.Data.BindingMode.TwoWay)

        End Sub
    End Class

End Namespace

File: 09/CS/CustomControls/09CustomControls/Client/UserCode/PasswordBoxControl901.cs
using System;
using System.Linq;
using System.IO;
using System.IO.IsolatedStorage;
using System.Collections.Generic;
using Microsoft.LightSwitch;
using Microsoft.LightSwitch.Framework.Client;
using Microsoft.LightSwitch.Presentation;
using Microsoft.LightSwitch.Presentation.Extensions;
using System.Windows;

namespace LightSwitchApplication
{
    public partial class PasswordBoxControl901
    {
        partial void PasswordBoxControl901_InitializeDataWorkspace(global::System.Collections.Generic.List<global::Microsoft.LightSwitch.IDataService> saveChangesTo)
        {
            // Write your code here.
            this.UserProperty = new User();
        }

        partial void PasswordBoxControl901_Saved()
        {
            // Write your code here.
            this.Close(false);
            Application.Current.ShowDefaultScreen(this.UserProperty);
        }

        partial void PasswordBoxControl901_Activated()
        {
            // Write your code here.

            var password = this.FindControl("Password");
            password.SetBinding(System.Windows.Controls.PasswordBox.PasswordProperty, "Value", System.Windows.Data.BindingMode.TwoWay );


        }
    }
}